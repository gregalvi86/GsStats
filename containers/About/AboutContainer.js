import React, { Component } from 'react'
import { About } from 'components'
import { updatePageInfo } from 'config/metadata'

class AboutContainer extends Component {
  componentDidMount() {
    const pageInfo = {
      title: 'about Â· uxscoreboard'
    }
    updatePageInfo(pageInfo)
  }
  render() {
    return <About />
  }
}

export default AboutContainer
