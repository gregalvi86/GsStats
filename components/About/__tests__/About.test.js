import React from 'react'
import { shallow } from 'enzyme'

import About from '../About'

describe('About.js', () => {
  it('renders correctly', () => {
    expect(shallow(<About />)).toMatchSnapshot()
  })
})
