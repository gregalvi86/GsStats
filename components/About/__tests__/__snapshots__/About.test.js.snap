// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.js renders correctly 1`] = `ShallowWrapper {}`;
